docs/source/contributing/translation.rst

```diff
@@ -0,0 +1,51 @@
+====================
+Internationalization
+====================
+
+If you want to help with translating the docs:
+
+#. Make sure you have a `Transifex <https://www.transifex.com/>`_ account. Transifex is a tool for colaborative translation and allows Manim team to review and help to improve suggestions from the community. You will use Transifex to submit your translation suggestions.
+#. Open to the `Manim project page <https://www.transifex.com/manim-community/manim-community-documentation>`_ .
+#. Click on Join Team.
+#. Select the language you want to help with.
+
+Congratulations ! You are now a Translator for the Manim Community project!
+
+
+You have access to the projects you joined just on the left of your profile picture
+on the top bar.
+
+.. image:: /_static/select_transifex_repo.png
+   :align: center
+   :width: 200
+
+Then you can go the
+`languages list <https://www.transifex.com/manim-community/manim-community-documentation/languages/>`_
+and select the language you want to help with. Then you have a blue button reading
+`Translate`, click it to get started with translating ! You will get to a list of
+all resources of the project. You can click on any untranslated resource, then
+click on `Translate` and you will get the following page:
+
+.. image:: /_static/transifex_explanation.png
+    :align: center
+
+The `A` panel is the list of all the strings in the resource. Click on any of
+them to translate it. It will appear in the `B` panel, and you will be able to
+set your translation in the `C` panel. Once you're done, click `Save changes`.
+Your translation will have to be reviewed before getting in the docs.
+You can also get back to the resources list by clicking `d` or change the
+destination language in `e`. Your
+
+
+If you want to become a reviewer, please join the discord server of Manim and
+you can ping the role @i18nService.
```

Comment:
Not sure if we need a specific role for this? I think we can just have moderators handle it.

Reply:
Well, it is about Transifex moderators that have to manually change the role of the translator to reviewer. Moderators on discord might not be moderators on Transifex.

Reply:
Fair enough, but I think a name like "Translation admin" would be better.